File: Samples/C#/FTP Sync/FtpSyncMain.cs
using System;

namespace FtpSync
{
	static class FtpSyncMain
	{
		[STAThread]
		static void Main(string[] args)
		{
			FtpSyncWorker lWorker = new FtpSyncWorker();

			if (lWorker.CheckArgs(args))
			{
				lWorker.Sync();
			}
		}
	}
}

File: Samples/VB/FTP Sync/FtpSyncMain.vb
Module FtpSyncMain
    Sub Main(ByVal args As String())
        Dim lWorker As FtpSyncWorker
        lWorker = New FtpSyncWorker
        If lWorker.CheckArgs(args) Then
            lWorker.Sync()
        End If
    End Sub
End Module
